source/conf.py

```diff
@@ -82,6 +82,7 @@
 .. _oneAPI:  https://oneapi.com
 .. _SYCL: https://www.khronos.org/sycl/
 .. |SYCL_SPEC| replace:: `SYCL Specification`_
+.. |SYCL_SPEC_HEADER_FILES| replace:: SYCL Specification `Section 4.3 <https://registry.khronos.org/SYCL/specs/sycl-2020-provisional.pdf#page=56&zoom=100,96,189>`__
```

Comment:
Same comment as in: https://github.com/KhronosGroup/SYCL_Reference/pull/62#discussion_r1340008363

Reply:
Updated links in the latest commit